src/plugins/custom_integrations/public/components/fleet_integration/elasticsearch_php/elasticsearch_php_readme.tsx

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+
+// eslint-disable-next-line @kbn/eslint/module_migration
+import styled from 'styled-components';
+import cuid from 'cuid';
```

Comment:
Just want to make it clear that the "generate API key" workflow we're using here is just a placeholder generating `cuid` values (https://www.npmjs.com/package/cuid). If we're intending to put these language clients in front of users we'll want to remove this, replace it with instructions to generate an API key, or add a component that does the API generation in an interactive way. 

Not sure if this on the language client team's radar.